### ISSUE TITLE
Ionic 4 with ngx-quill works fine (not issue)

### ISSUE THREAD

**Issue by kasgaormeo** [KillerCodeMonkey/ngx-quill#335]
Hello, KillerCodeMonkey!

This is Kasgaromeo. Your ngx-quill is working really nice in my application!
I really appreciated it!

Only Syntax not working yet with both HighlightJS and NGX-Highlight.. 

Great Job!

![ngxionic](https://user-images.githubusercontent.com/33588040/52528867-99f2b180-2cb6-11e9-9c8f-7f635629b435.jpeg)

**Comment by KillerCodeMonkey** [comment]
Did you checked out my example Repo? There i have an example for Syntax
highlight.

You need to add Highlightjs to your Scripts. And Set a color Schema Like
"Monokai sublime" and configure Highlightjs which code you want to Support.

I have done this in the index.html.

**Comment by kasgaormeo** [comment]
I did it. But, it didn't work on IOS.

I will try again tomorrow.

Thanks for your quick response.

Best Regard,

**Comment by KillerCodeMonkey** [comment]
and it is working on a non mobile browser and android?

**Comment by kasgaormeo** [comment]
Yes. It is perfectly working on my admin site, but I don’t know Android... your ngx-quill should work on native side of Android because it works on my Iphone and Ipad as well. Not syntax though. I will try tomorrow.

Thanks,

**Comment by KillerCodeMonkey** [comment]
Ah okay, sounds like a highlight.js thing.

i will close this issue, but feel free to let us know, if and how you solved it :).

Thanks for the nice feedback. :)

**Comment by KillerCodeMonkey** [comment]
@kasgaormeo a little idea, does this happen when you build your app for ios and did you use a cdn to load hightlightjs?

Maybe you get some problems with the iOS security policies because you are trying to load from external resources. You can configure that, that your app needs internet access and is allowed to access the cdn.

**Comment by KillerCodeMonkey** [comment]
i added an ionic v4 example:
https://github.com/KillerCodeMonkey/ngx-quill-ionic-v4

It is working on android simulator with javascript syntax highlighting. 
iOS simulator is lagging like hell on my macbook, so maybe you can test it :).

**Comment by kasgaormeo** [comment]
Hi, KillerCodeMoneky.

Finally, it works on my iPhone with several tests.
Here is screenshot.

![ionicsuccess](https://user-images.githubusercontent.com/33588040/52536535-dc090b00-2d29-11e9-967c-1b4ee3536c28.jpeg)

So, inside of <quill-editor>, syntax directive is not working. So, I tried changing like you as declaring in the module with forRoot({ syntax: true }). Finally, it works with Cordova with Desktop.

However, it wasn't working for my IOS. So, I tried to change CDN with https and no https => it didn't work.

Finally, I installed highlight.js with NPM and declare only highlight.js's CSS file into global.scss.

And it works as natively! 

Thanks, man.

I appreciated your hard work!
